homeassistant/components/zha/entity.py

```diff
@@ -202,9 +202,12 @@ def async_restore_last_state(self, last_state) -> None:
 
     async def async_update(self) -> None:
         """Retrieve latest state."""
+        tasks = []
```

Comment:
```suggestion
```suggestion
    tasks = [
        channel.async_update()
        for channel in self.cluster_channels.values()
        if hasattr(channel, "async_update")
    ]
```
```